entrypoint.sh

```diff
@@ -9,6 +9,11 @@ sleep 10
 # Pull the model
 /usr/bin/ollama pull llama3.2:3b
 
+if [ $? -ne 0 ]; then
+  echo "Model pull failed"
+  exit 1
+fi
```

Comment:
This is another way of doing the same 

```suggestion 
if ! /usr/bin/ollama pull llama3.2:3b; then
  echo "Model pull failed"
  exit 1
fi
```